### ISSUE TITLE
"TypeError: toolbar is null" on QUnit v1.17

### ISSUE THREAD

**Issue by UYEONG** [karma-runner/karma-qunit#23]
- karma-qunit v0.1.4
- qunitjs v1.17.0
### TEST CODE

``` javascript
QUnit.module( "module A", {
    beforeEach: function() {
        // prepare something for all following tests
    },
    afterEach: function() {
        // clean up after each test
    }
});

QUnit.test('some test', function(assert) {
    assert.equal(1, 1);
});
```
### RESULT

```
TypeError: toolbar is null
```
### QUNIT CODE

``` javascript
function toolbarModuleFilter() {
    var toolbar = id( "qunit-testrunner-toolbar" ), // <= find element.
        moduleFilter = document.createElement( "span" ),
        moduleFilterHtml = toolbarModuleFilterHtml();

    if ( !moduleFilterHtml ) {
        return false;
    }

    moduleFilter.setAttribute( "id", "qunit-modulefilter-container" );
    moduleFilter.innerHTML = moduleFilterHtml;

    addEvent( moduleFilter.lastChild, "change", applyUrlParams );

    toolbar.appendChild( moduleFilter ); // <= error!
}
```

**Comment by jlonardi** [comment]
Having the same problem. Is it possible to specify the qunit version karma uses?

**Comment by UYEONG** [comment]
You can specify the version of qunitjs when you install. And FYI, the issue 
doesn't occur if you use qunitjs v1.16.

**Comment by JamesMGreene** [comment]
Sorry about that! We fixed this regression in QUnit 1.17.1: https://github.com/jquery/qunit/commit/c0d9ad6cfc73157b03bc9bec5b0aee875150b5aa

**Comment by Krinkle** [comment]
@dignifiedquire This can be closed now. It was an issue in upstream QUnit 1.17.0, fixed years ago in QUnit 1.17.1.
